dropshot/src/config.rs

```diff
@@ -113,9 +116,108 @@ impl Default for ConfigDropshot {
     fn default() -> Self {
         ConfigDropshot {
             bind_address: "127.0.0.1:0".parse().unwrap(),
-            request_body_max_bytes: 1024,
+            default_request_body_max_bytes: 1024,
             default_handler_task_mode: HandlerTaskMode::Detached,
             log_headers: Default::default(),
         }
     }
 }
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(default)]
+struct DeserializedConfigDropshot {
+    bind_address: SocketAddr,
+    default_request_body_max_bytes: usize,
+    // Previous name for default_request_body_max_bytes. Present only to guide
+    // users to the new name.
+    #[serde(
+        deserialize_with = "deserialize_invalid_request_body_max_bytes",
+        skip_serializing_if = "Option::is_none"
```

Comment:
I think this could be simply `skip_serializing` in that it has to be None

Reply:
Ah I didn't know `skip_serializing` was a thing! Makes sense.